packages/contracts-bedrock/src/cannon/MIPS.sol

```diff
@@ -183,7 +183,7 @@ contract MIPS is ISemver {
                 });
                 (v0, v1, state.preimageOffset, state.memRoot,,) = sys.handleSysRead(args);
             } else if (syscall_no == sys.SYS_WRITE) {
-                (v0, v1, state.preimageKey, state.preimageOffset) = sys.handleSysWrite({
+                sys.SysWriteParams memory args = sys.SysWriteParams({
```

Comment:
Does it work if you pass the struct directly into the function call instead of creating a variable?

Reply:
yeah

Reply:
do you recommend it's done that way rather? that's actually how i did it in the initial code coverage pr

Reply:
Yeah let's do it that way

Reply:
Nvm since this matches the existing pattern it's ok as-is